zeekpkg/package.py

```diff
@@ -57,6 +57,15 @@ def is_valid_name(name):
     return True
 
 
+def validate_aliases(metadata_dict):
```

Comment:
Do we need a new validator here, or could we instead require that `alias` is a valid name and extend the `is_valid_name` validator? It seems that an alias would need to be a valid package name.